specification/summary-care-record.yaml

```diff
@@ -437,8 +437,10 @@ components:
         The user role ID (URID) for the current session. Also known as a user role profile ID (URPID).
 
         For more details, see [determine the user’s role](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/user-restricted-restful-apis-nhs-identity-combined-authentication-and-authorisation#step-6-determine-the-user-s-role).
+        
+        This field is optional. 
 
-      required: true
+      required: false
```

Comment:
What I meant in my previous comment - this field is not optional in SCR API because business logic requires it. We should remove the "This field is optional. " description and change the flag back to "required"

Reply:
@dawidskainos It's optional because the `userrole` shared flow that we've added falls back to the userrole that's inside the token. So if you pass `NHSD-Session-URID` that'll takes precedence over `selected_userrole` claim inside token. If not (hence optional),  it falls back to your token. And if it can't find the role at all then, it will throw error.

Reply:
@artronics thanks, that makes sense